demos/workflows/portfolio/README.md

```diff
@@ -38,6 +38,8 @@ This demo shows how to use Maestro to create a portfolio management system.
 ```bash
 cd maestro
 uv pip install -e .
+uv pip install yfinace
+uv pip install qiskit-ibm-catalog
```

Comment:
I think this is fine for now, but longer-term adding something like this to the pyproject.toml might be better

```toml
[project.optional-dependencies]
demo_portfolio = [
  "yfiance",
  "qiskit-ibm-catalog"
]
```

then you could install with something like `uv pip install .[demo_portfolio]`

Reply:
What about separate `requirements-demos.txt` file?

What's the best practice?

Reply:
if you're using pyproject.toml, I think best practice is to put all dependencies in there rather than having an additional requirements.txt file -- that will let you also reconcile the dep versions between main and demo(s) too, which you wouldn't get with multiple files

I can look into making a PR for that tomorrow (unless someone else really wants to tackle it)

Reply:
> if you're using pyproject.toml, I think best practice is to put all dependencies in there rather than having an additional requirements.txt file -- that will let you also reconcile the dep versions between main and demo(s) too, which you wouldn't get with multiple files

I'm actually actively working on this change rn as part of #506

Reply:
I'll leave this to Alex handling in #506